dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -0,0 +1,181 @@
+    xmlns:local1="using:MUXControlsTestApp.Scripts">
+
+
+    <local1:Int32ToDoubleConverter x:Key="IntToDouble"/>
+    <local1:IntToStrConverter x:Key="IntToStr"/>
+    <local1:Int32MinusOneConverter x:Key="IntMinusOne"/>
+
+    <DataTemplate x:Key="IntToButton">
+        <Button Content="{Binding}" MinWidth="35"
+                Background="White"
+                Foreground="Black"/>
+    </DataTemplate>
+
+    <Style TargetType="local:PrototypePager">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:PrototypePager">
+                    <Grid
+                        Background="{TemplateBinding Background}"
+                        BorderBrush="{TemplateBinding BorderBrush}"
+                        BorderThickness="{TemplateBinding BorderThickness}">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto"/>
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*"/>
+                            <ColumnDefinition Width="Auto"/>
+                            <ColumnDefinition Width="*"/>
+                        </Grid.ColumnDefinitions>
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="FirstPageButtonStates">
+                                <VisualState x:Name="FirstPageButtonVisible"/>
+                                <VisualState x:Name="FirstPageButtonCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="FirstPageButton.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="PreviousPageButtonStates">
+                                <VisualState x:Name="PreviousPageButtonVisible"/>
+                                <VisualState x:Name="PreviousPageButtonCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="PreviousPageButton.Visibility" Value="Collapsed"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="NextPageButtonStates">
+                                <VisualState x:Name="NextPageButtonVisible"/>
+                                <VisualState x:Name="NextPageButtonCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="NextPageButton.Visibility" Value="Collapsed"/>
+                                        <Setter Target="NextPageButton.IsEnabled" Value="False"/>
```

Comment:
Why do you disable this one but not the Previous and First?

Reply:
Sorry, forgot to remove these lines. They have been removed